index.html

```diff
@@ -57,10 +57,10 @@
             </div>
             <!-- When you add more languages do it in the above format-->
         </div>
-        <a href="https://github.com/fossasia/labyrinth">
+          
           <h1 class="brand-logo center">Labyrinth</h1>
-        </a>
-        <ul class="right hide-on-med-and-down">
+        
+          <ul class="right hide-on-med-and-down">
```

Comment:
I think, the indentation is a little off.
i.e. the ul could be two spaces to the left, so the li in it is indented.
Good indentation helps developing fast.
Maybe the h1 above it was wrong, I do not know.

Reply:
@niccokunzmann Thanks for the suggestion. I will correct it take care of it in future.